---
title: ""
description: "The Black Death, Epigenomic MERFISH, Single-Cell Long-Read Sequencing, Be Mindful of the Defaults, and more"
date: "2022-10-16"
categories: [natural selection, epigenomics, annotation, MAS-Seq]
format:
    html:
        self-contained: true
draft: true
---

## Applications

The Black Death left its mark not only on our history but also on our immunity. Luis Barreiro's lab showed that **natural selection driven by the Black Death, that were protective at that time, could be linked to present-day susceptibility to autoimmune diseases, such as Crohn's disease.**^[https://www.nature.com/articles/s41586-022-05349-x#Fig3] Great combinations of ancient DNA work, population genomics, and in-vitro infections.

Nicolet and Wolkers showed that **mRNA expression levels are poorly correlated with protein abundance (Pearson r ~ 0.4)**.^[https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0276294#sec015] Interestingly, certain gene charactistics, such as untranslated region (UTR) lengths, could affect mRNA-protein correlation.

::: {.column-margin}
![Figure 1E: Correlation between mRNA and protein abundance in activated T cells.](images/paste-CA6B1DB9.png)
:::

## Methods

Cell-type annotation is a critical step in single-cell analysis pipeline. For scATAC-Seq, we can annotate cells using either tools developed for scRNA-Seq or transfer labels from RNA to ATAC. Here, Yuge Wang and colleagues **performed a thorough benchmarking and showed subpar performance of all these methods.**[^1] Using Multiome data as reference can help improve the accuracy, but only to some extent.

[^1]: https://www.biorxiv.org/content/10.1101/2022.10.05.511014v1

::: column-margin
![Weighted accuracy of cell-type prediction. Note the \<50% accuracy for all methods.](images/paste-F0EF8E4B.png)
:::

## Technologies

Xiaowei Zhuang published Epigenomic MERFISH, an **imaging-based technology for spatially resolved single-cell epigenomic profiling.**[^2] Epigenomic MERFISH maps active promoters and enhancers in mouse brain and reveals putative enhancer hubs in embryonic mouse brain. 

::: {.column-margin}
Awesome spatial technologies seem to be published on a weekly basis. I wonder which ones will be commercialized and used around the world?
:::

[^2]: https://www.cell.com/action/showPdf?pii=S0092-8674%2822%2901254-5

![Epigenomic MERFISH](images/paste-3CE397CB.png)

PacBio, in partnership with the Broad Institute and 10X Genomics, launched the new Multiplexed Arrays Sequencing (MAS-Seq).[^3] This technology **enables long-read scRNA-Seq for identification of alternatively spliced isoforms and gene fusions without the need for *de novo* assembly.**[^4]

[^3]: https://www.pacb.com/press_releases/pacbio-launches-single-cell-transcriptome-solution-ushering-in-a-new-era-of-rna-discovery/

[^4]: https://www.biorxiv.org/content/10.1101/2021.10.01.462818v1

::: column-margin
Excellent move by 10X's single-cell division! Looking forward to the maneuvers of its spatial division.
:::

## Conversations

**Inter-operability of single-cell analysis frameworks is not enough, the tools should be coherent as well!** Log-fold change calculation in Seurat and Scanpy differs significantly^[https://twitter.com/slavov_n/status/1582347828818456576], as pointed out by Nikolai Slavov from a recent benchmarking paper by Davis McCarthy's lab.^[https://www.biorxiv.org/content/10.1101/2022.05.09.490241v2] This mess begs for a question: **to whom should we handle the responsibility for making the decisions in single-cell data analysis? Developers or Users?**

Related to the post above, Nicola RomanÃ² shared his observation on the danger of default parameters in single-cell analysis frameworks.^[https://twitter.com/_nicoromano_/status/1582276701874098176] If you're using Seurat, **remember to pass your `seed` into its function** instead of declaring it before calling the function, as per usual, **or else the function will override your seed.**

Yet another thread on the danger of (blindly following) the defaults, **Junhyong Kim shared his thoughts on Seurat's CP10K normalization** (i.e. dividing gene count by library depth, multiply by 10K, followed with `log1p`).^[https://twitter.com/JunhyongKim/status/1582421723286106112]. Related to this, Lior Pachter's lab published a benchmark on normalisation methods and suggested proportional fitting --> log-transformation --> proportional fitting (PFlog1pPF).^[https://www.biorxiv.org/content/10.1101/2022.05.06.490859v1] Or should we abandon the normalisation route and instead fully embrace the count nature of scRNA-Seq data?^[https://www.biorxiv.org/content/10.1101/2022.06.01.494334v1.full]

:::{.column-margin}
Getting confused now? Remember to include bioinformatician(s) in your team from the start.
:::

## Miscellaneous

### Jobs

Luciano Martelotto has an [open Senior Scientist position](https://www.seek.com.au/job/58876017?savedSearchID=bcd60b5c-7ce8-11e9-8e48-8f51da720424).

## Acknowledgements {.appendix}

Credits to the original authors of the Twitter threads. I am here only to curate stuffs :)
